armi/reactor/converters/uniformMesh.py

```diff
@@ -1416,45 +1420,63 @@ def __init__(self, reactorParamNames, blockParamNames, b):
         self.reactorParamNames = reactorParamNames
         self.blockParamNames = blockParamNames
 
-    @staticmethod
-    def paramSetter(block, vals, paramNames):
+    def paramSetter(self, block: "Block", vals, paramNames):
         """Sets block parameter data."""
         for paramName, val in zip(paramNames, vals):
             # Skip setting None values.
             if val is None:
                 continue
 
             if isinstance(val, (tuple, list, np.ndarray)):
-                ParamMapper._arrayParamSetter(block, [val], [paramName])
+                self._arrayParamSetter(block, [val], [paramName])
             else:
-                ParamMapper._scalarParamSetter(block, [val], [paramName])
+                print(paramName)
+                self._scalarParamSetter(block, [val], [paramName])
 
-    def paramGetter(self, block, paramNames):
+    def paramGetter(self, block: "Block", paramNames):
         """Returns block parameter values as an array in the order of the parameter names given."""
         paramVals = []
+        symmetryFactor = block.getSymmetryFactor()
         for paramName in paramNames:
+            multiplier = self.getFactorSymmetry(paramName, symmetryFactor)
             val = block.p[paramName]
             # list-like should be treated as a numpy array
-            if isinstance(val, (tuple, list, np.ndarray)):
-                paramVals.append(np.array(val) if len(val) > 0 else None)
+            if val is None:
+                if isinstance(val, (tuple, list, np.ndarray)):
+                    paramVals.append(np.array(val) if len(val) > 0 else None)
+                else:
+                    paramVals.append(val)
```

Comment:
I'm confused about the need for this whole block. If `val is None`, then it's not going to be a `tuple`, `list`, or `np.ndarray`.

I think one layer of the `if`/`else` could be removed here.

```
            if isinstance(val, (tuple, list, np.ndarray)):
                paramVals.append(np.array(val) * multiplier if len(val) > 0 else None)
            elif val is not None:
                paramVals.append(val * multiplier)
            else:
                paramVals.append(val)
```